00 - Meta/00 - Meta.md

```diff
@@ -0,0 +1,18 @@
+---
+alias:
```

Comment:
I know that `alias` also works, but in the docs we only say to use `aliases`, so we shouldn't confuse contributors.
https://help.obsidian.md/How+to/Add+aliases+to+note#Set+aliases